README.md

```diff
@@ -50,12 +50,21 @@ The Tabulator produces the following as output:
     
     `$ chmod 777 gradlew`
 
+#### Encrypting the Tabulator Directory
+For security purposes, we **strongly recommend** applying password encryption (e.g. 256-bit SHA) to the directory containing the Tabulator, config files, CVR files, and any other related files.
+
+We recommend using open-source utilities such as [7-Zip](https://www.7-zip.org/) for Windows or EncFS, gocryptfs, etc. for Linux (see [this comparison](https://nuetzlich.net/gocryptfs/comparison/)). 
+
+Mac OS has built-in encryption capability that allows users to create encrypted disk images from folders using Disk Utility (see ["Create a secure disk image"](https://support.apple.com/guide/disk-utility/create-a-disk-image-dskutl11888/mac)). 
+
 ## Configuring a Contest
 
 The GUI can be used to easily create, save, and load contest configuration files (which are in .json format). These files can also be created manually using any basic text editor, but this method isn't recommended.
 
 In either case, please reference the [config file documentation](src/main/resources/network/brightspots/rcv/config_file_documentation.txt) when configuring a contest.
 
+**Warning**: Avoid clicking in the command prompt / terminal window when starting the Tabulator GUI, as it may halt the startup process.
```

Comment:
```suggestion
**Warning**: using system shortcuts, aliases, and symbolic links are not recommended and not supported.  Doing so may result in unexpected behavior.
```

Reply:
Not entirely convinced this error only occurs when shortcuts are used, so I'll include both bits of this language to cover our bases.